header.go

```diff
@@ -723,23 +670,23 @@ func (h *RequestHeader) SetMethodBytes(method []byte) {
 }
 
 // Protocol returns HTTP protocol.
-func (h *RequestHeader) Protocol() []byte {
-	if len(h.proto) == 0 {
+func (h *header) Protocol() []byte {
+	if len(h.protocol) == 0 {
 		return strHTTP11
 	}
-	return h.proto
+	return h.protocol
 }
 
 // SetProtocol sets HTTP request protocol.
 func (h *RequestHeader) SetProtocol(method string) {
```

Comment:
Can you change the param name to `protocol` while you're making changes here anyways 😄